Packs/FlashpointFeed/Integrations/IgniteFeed/IgniteFeed.py

```diff
@@ -0,0 +1,597 @@
+        next_run['next_updated_since'] = fetch_params['updated_since']
+        next_run['next_updated_until'] = fetch_params['updated_until']
+        # Set only offset equal to previous offset + max_fetch.
+        next_run['offset'] = next_run.get('offset', DEFAULT_OFFSET) + MAX_FETCH
+
+    indicators = client.create_indicators_from_response(response=response, params=params)
+
+    demisto.debug(f"Set the last Run for indicators: {next_run}")
+
+    return indicators, next_run
+
+
+def flashpoint_ignite_get_indicators_command(client: Client, params: dict, args: dict) -> CommandResults:
+    """
+    Get limited number of indicators.
+
+    :param client: Client object.
+    :param params: Dictionary of parameters.
+    :param args: Dictionary of arguments.
+
+    :return: Standard Command Result.
+    """
+    fetch_params = validate_get_indicators_args(args=args)
+
+    response = client.fetch_indicators(params=fetch_params)
+
+    response = remove_empty_elements(response)
+
+    indicators = client.create_indicators_from_response(response=response, params=params)
+
+    if not indicators:
+        return CommandResults(readable_output=MESSAGES['NO_INDICATORS_FOUND'])
+
+    readable_output = prepare_hr_for_indicators(indicators=indicators, platform_url=client.platform_url)
+    return CommandResults(
+        readable_output=readable_output,
+        raw_response=indicators
+    )
+
+
+'''Main Function'''
+
+
+def main():
+    """Parse params and runs command functions."""
+    params = remove_space_from_args(demisto.params())
+    remove_nulls_from_dictionary(params)
+
+    # Get the service API url
+    base_url = params.get('url', DEFAULT_API_PATH)
+
+    api_key = str(params.get('credentials', {}).get('password', '')).strip()
+
+    # Default configuration parameters for handling proxy and SSL Certificate validation.
+    insecure = not argToBoolean(params.get("insecure", False))
+    proxy = argToBoolean(params.get("proxy", False))
+
+    command = demisto.command()
+    demisto.debug(f'[Ignite] Command being called is {command}')
+
+    try:
+        validate_params(params=params)
+        headers: Dict = {
+            'Authorization': f'Bearer {api_key}',
+            'X-FP-IntegrationPlatform': INTEGRATION_PLATFORM,
+            'X-FP-IntegrationPlatformVersion': get_demisto_version_as_str(),
+            'X-FP-IntegrationVersion': INTEGRATION_VERSION
+        }
+        client = Client(
+            verify=insecure,
+            proxy=proxy,
+            url=base_url,
+            headers=headers
+        )
+        args = demisto.args()
+        if command == 'test-module':
+            return_results(test_module(client=client))
+        elif command == 'fetch-indicators':
+            last_run = demisto.getLastRun()
+            indicators, next_run = fetch_indicators_command(client=client, params=params, last_run=last_run)
+            demisto.setLastRun(next_run)
+            demisto.createIndicators(indicators)
```

Comment:
How many indicators are supposed to be fetched from the API in each fetch? it may be better to use batches when creating indicators:

for b in batch(indicators, batch_size=2000):
                demisto.createIndicators(b)

Reply:
We're creating 1000 indicators.